nhrpd/linux.c

```diff
@@ -32,6 +32,11 @@
 #include "os.h"
 #include "netlink.h"
 
+#ifndef HAVE_STRLCPY
+size_t strlcpy(char *__restrict dest,
+               const char *__restrict src, size_t destsize);
+#endif
+
```

Comment:
> 
> this can be avoided by including "zebra.h" instead.
> but you can keep it as is for now.

Code does not actually compile when zebra.h is included, because zebra definitions for a bunch of things clash with the values brought in from the kernel header files. Resolving all that seemed not worth the effort, so I just put the forward declaration for the function there directly.

Reply:
this is fine. thanks for having tried.